### ISSUE TITLE
question about the shipping label of sandbox store

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9844]
We get the shipping label c2ad6910-6376-11ef-97df-7166871624c9  from **/shipment-management/label**, but it looks very strange(
[2024-09-09_13-54-10.pdf](https://github.com/user-attachments/files/16930006/2024-09-09_13-54-10.pdf)
) . 
Why it looks like this? In the real environment also like this?

**Comment by PrzemyslawLukanowski** [comment]
It looks like this only on Sandbox, on the production environment it will be a proper label.

**Comment by yangcen6** [comment]
Could you help me transfer the logistics track of this order to a successful delivery?

**Comment by PrzemyslawLukanowski** [comment]
Yes, I added successful delivery tracking to this parcel: A0000467B6. You can check it on the website from the buyer's side and as a seller, by using GET /order/carriers/ALLEGRO/tracking?waybill=A0000467B6.

**Comment by yangcen6** [comment]
Can you provide us the shipping label in production environment?

**Comment by PrzemyslawLukanowski** [comment]
For example:
<img width="591" alt="Zrzut ekranu 2024-09-16 o 07 33 45" src="https://github.com/user-attachments/assets/de30aea5-9675-4422-94b9-a26fcd2408b7">
